packages/dd-trace/src/appsec/reporter.js

```diff
@@ -25,11 +53,10 @@ const library = {
   lib_version: libVersion
 }
 
+const toFinish = new WeakMap()
```

Comment:
Please add a comment to explain what this one is used for

Reply:
```suggestion
// this is the list of "unfinished" events awaiting response data population before being sent for each gateway context
const toFinish = new WeakMap()
```